packages/ptmap/ptmap.2.0.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "Jean-Christophe.Filliatre@lri.fr"
+authors: ["Jean-Christophe Filliâtre"]
+homepage: "https://www.lri.fr/~filliatr/software.en.html"
+bug-reports: "https://github.com/UnixJunkie/ptmap/issues"
+dev-repo: "https://github.com/UnixJunkie/ptmap.git"
+license: "GPL-2.1"
+build: [
+  ["obuild" "configure"]
+  ["obuild" "build" "lib-ptmap"]
+]
+install: [
+  ["obuild" "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "ptmap"]
+]
+depends: [
+  "ocamlfind"
```

Comment:
I'm not even sure it's needed for the build.
I'm just sure it's needed for the uninstall step.